doc/classes/AnimationTreePlayer.xml

```diff
@@ -423,6 +429,7 @@
 			<argument index="2" name="enable" type="bool">
 			</argument>
 			<description>
+				If [code]enable[/code] is [code]true[code], the oneshot node [code]id[/code] assigns an inherited weight of [code]0[/code] to the node at relative [NodePath] [code]path[/code].
```

Comment:
Similar to [Comment No. 4]

Reply:
Also take a  look at [Comment No. 3]

Reply:
Oh, yeah, I rewrote something that looked like that for a different description...I'll fix that. The "binds" thing too.